hera_sim/noise.py

```diff
@@ -7,26 +7,28 @@
 import aipy
 import os
 from .data import DATA_PATH
-HERA_TSKY_VS_LST_NPZ = os.path.join(DATA_PATH, 'HERA_Tsky_vs_LST.npz')
-
-npz = np.load(HERA_TSKY_VS_LST_NPZ) # Tsky vs fq/lst from Beardsley, beam v XXX, GSM v XXX
-fqs = npz['freqs'] / 1e3
-lsts = npz['lsts'] / 12. * np.pi
-lsts = np.concatenate([lsts[-10:]-2*np.pi, lsts, lsts[:10]+2*np.pi])
-HERA_Tsky_xx = npz['HERA_Tsky'][0].T
-HERA_Tsky_yy = npz['HERA_Tsky'][1].T
-HERA_Tsky_xx = np.concatenate([HERA_Tsky_xx[-10:], HERA_Tsky_xx, HERA_Tsky_xx[:10]])
-HERA_Tsky_yy = np.concatenate([HERA_Tsky_yy[-10:], HERA_Tsky_yy, HERA_Tsky_yy[:10]])
+from .interpolators import Tsky, _read_npy
+from .defaults import _defaults
+
+HERA_TSKY_VS_LST_NPZ = os.path.join(DATA_PATH, 'HERA_Tsky_Reformatted.npz')
+
 HERA_Tsky_mdl = {}
-HERA_Tsky_mdl['xx'] = RectBivariateSpline(lsts, fqs, HERA_Tsky_xx, kx=4, ky=4)
-HERA_Tsky_mdl['yy'] = RectBivariateSpline(lsts, fqs, HERA_Tsky_yy, kx=4, ky=4)
+HERA_Tsky_mdl['xx'] = Tsky(HERA_TSKY_VS_LST_NPZ, pol='xx')
```

Comment:
I want these kinds of lines removed completely from the code: the code itself shouldn't care about HERA-specific stuff. That should all live in the config module.